openstates/scrape/base.py

```diff
@@ -122,6 +126,36 @@ def __init__(
             handler = importlib.import_module(modname)
             self.scrape_output_handler = handler.Handler(self)
 
+    def push_to_queue(self):
+        """Push this output to the sqs for realtime imports."""
+
+        # Create SQS client
+        sqs = boto3.client("sqs")
+
+        queue_url = settings.SQS_QUEUE_URL
+        bucket = settings.S3_REALTIME_BASE.replace("s3://", "")
+
+        message_body = json.dumps(
+            {
+                "file_path": self.output_file_path,
+                "bucket": bucket,
+                "jurisdiction_id": self.jurisdiction.jurisdiction_id,
+            }
+        )
+
+        # Send message to SQS queue
+        response = sqs.send_message(
+            QueueUrl=queue_url,
+            DelaySeconds=10,
+            MessageAttributes={
+                "Title": {"DataType": "String", "StringValue": "S3 Output Path"},
+                "Author": {"DataType": "String", "StringValue": "Open States"},
+                "WeeksOn": {"DataType": "Number", "StringValue": "6"},
```

Comment:
Can we explain this number? `6` isn't very clear on its own.

Reply:
> Can we explain this number? `6` isn't very clear on its own.

We don't actually need it, got this from the official doc but it's a sample from the docs, I thought it was important initially